docs/en/concepts-and-designs/lal.md

```diff
@@ -0,0 +1,216 @@
+# Log Analysis Language
+
+Log Analysis Language (LAL) in SkyWalking is essentially a Domain-Specific Language (DSL) to analyze logs. You can use
+LAL to parse, extract, filter, analyze, and save the logs, as well as collaborate the logs with traces, and metrics.
```

Comment:
`parse, extract, filter, analyze, and save` VS `parser, extractor, and sink`. It makes me feel a little inconsistent.

Reply:
> `parse, extract, filter, analyze, and save` VS `parser, extractor, and sink`. It makes me feel a little inconsistent.

Fixed